packages/orchestration/src/exos/chain-hub.js

```diff
@@ -722,6 +722,24 @@ export const makeChainHub = (
       });
     },
 
+    /**
+     * @param {string} partialId CAIP-10 account ID or a Cosmos bech32 address
+     * @returns {AccountId}
+     * @throws {Error} if chain info not found for partialId
+     */
+    makeCaipAddress(partialId) {
```

Comment:
this looks to be exactly the same as the extant `makeChainAddress`. That is the canonical address type. We don't need to say "CAIP" everywhere because it's all going to be CAIP. Non-CAIP is legacy.

please remove this redundant function.

Reply:
done